tools/compile_clang.sh

```diff
@@ -54,6 +54,22 @@ WARN_PACKAGE () {
     fi
 }
 
+DOWNLOAD_HELPER() {
+    WGET_CTR=1
+    while [ 1 ]; do
+        wget --no-dns-cache --tries=3 --retry-connrefused --waitretry=3 $1 >/dev/null 2>&1
```

Comment:
Is there a reason you have omitted `--quiet`?

Reply:
Because it's not used.

Reply:
Well, I would imagine that we either use `wget --quiet` or `wget > /dev/null 2>&1`, we have just opted for the latter here?